crates/cxx-qt-lib/src/core/qdatetime.cpp

```diff
@@ -145,5 +145,15 @@ qdatetimeToSecsSinceEpoch(const QDateTime& datetime)
   return static_cast<::std::int64_t>(datetime.toSecsSinceEpoch());
 }
 
+void
+qdatetimeSetTimeZone(QDateTime& datetime, const QTimeZone& timeZone)
+{
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 7, 0))
+  datetime.setTimeZone(timeZone, TransitionResolution::LegacyBehavior);
```

Comment:
See the error, i suspect this needs to be `QDateTime::TransitionResolution::LegacyBehavior` ? 

https://code.qt.io/cgit/qt/qtbase.git/tree/src/corelib/time/qdatetime.h#n341

Reply:
yep it seems. I will adapt patch.

Reply:
done. Need to retest it.

Reply:
clang-format is now sad :-) looks like the line might be too long...